proto/substrait/algebra.proto

```diff
@@ -208,7 +208,7 @@ message AggregateRel {
   // A list of expression grouping that the aggregation measured should be calculated for.
   repeated Grouping groupings = 3;
 
-  // A list of one or more aggregate expressions along with an optional filter.
+  // A list of one or more aggregate expressions along with an optional filter, and optional ordering.
```

Comment:
The ordering isn't part of the measures is it?

Reply:
Actually, it is part of the measure similar to the filter.

Reply:
Since `AggregateRel` stores `repeated Measure measures` for each aggregate expression. Adding a new measure type would require us to store another `repeated Measure2 measures2`. I think, it is a bit confusing. However, it is an option also. I am not sure which approach is the right one. What do you think?

Reply:
I think `measures` is the correct place for this.  There is one measure per output column.  Each output column could expect a different ordering for its inputs.  This is consistent for how this is expressed in SQL.